packages/driver/cypress/fixtures/errors.html

```diff
@@ -54,7 +54,7 @@
     document.querySelector(".trigger-async-error").addEventListener('click', function () {
       setTimeout(function () {
         one('async error')
-      }, 500)
+      }, 0)
```

Comment:
This has only been clicked by a single test (before my PR) and the 500 timeout wasn't really required for it to pass or anything. So I've decreased the time to speed up the test a little bit and avoid a test timeout in the test that I've added 😉